### ISSUE TITLE
Alpine Linux support ?

### ISSUE THREAD

**Issue by tommywalkie** [tommywalkie/excalidraw-cli#12]
From `node-canvas` ' **Installation** section :

> By default, binaries for macOS, Linux and Windows will be downloaded. If you want to build from source, use `npm install --build-from-source and see the **Compiling** section below.

This means `excalidraw-cli` (and `node-canvas`) will work out of the box for macOS, Ubuntu and Windows. 

However, Alpine Docker images will require some modules in order to make the Cairo backend dependency work.

**Comment by tommywalkie** [comment]
Found the proper configuration. 👍 

```dockerfile
FROM node:alpine

RUN apk update
RUN apk add --no-cache \
    python \
    g++ \
    build-base \
    cairo-dev \
    jpeg-dev \
    pango-dev \
    musl-dev \
    giflib-dev \
    pixman-dev \
    pangomm-dev \
    libjpeg-turbo-dev \
    freetype-dev
```

Just need to document this somewhere on the README and can close the issue.

**Comment by tommywalkie** [comment]
Solved with #13
